target/linux/ath79/dts/qcn5502_netgear_ex7300-v2.dts

```diff
@@ -1,255 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
-#include "qca956x.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
+#include "qcn5502_netgear_ex7300-v2.dtsi"
 
 / {
 	model = "Netgear EX7300 v2";
-	compatible = "netgear,ex7300-v2", "qca,qcn5500", "qca,qca9560";
-
-	aliases {
-		led-boot = &led_power_green;
-		led-failsafe = &led_power_amber;
-		led-running = &led_power_green;
-		led-upgrade = &led_power_amber;
-		label-mac-device = &eth0;
-	};
-
-	led_spi {
-		compatible = "spi-gpio";
-		#address-cells = <1>;
-		#size-cells = <0>;
-
-		sck-gpios = <&gpio 14 GPIO_ACTIVE_HIGH>;
-		mosi-gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
-		num-chipselects = <0>;
-
-		led_gpio: led_gpio@0 {
-			compatible = "fairchild,74hc595";
-			reg = <0>;
-			gpio-controller;
-			#gpio-cells = <2>;
-			registers-number = <1>;
-			spi-max-frequency = <500000>;
-		};
-	};
-
-	leds {
-		compatible = "gpio-leds";
-
-		led_power_green: power_green {
-			label = "green:power";
-			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
-		};
-
-		led_power_amber: power_amber {
-			label = "amber:power";
-			gpios = <&led_gpio 7 GPIO_ACTIVE_LOW>;
-		};
-
-		wps_green {
-			label = "green:wps";
-			gpios = <&led_gpio 6 GPIO_ACTIVE_LOW>;
-		};
-
-		router_red {
-			label = "red:router";
-			gpios = <&led_gpio 5 GPIO_ACTIVE_LOW>;
-		};
-
-		router_green {
-			label = "green:router";
-			gpios = <&led_gpio 4 GPIO_ACTIVE_LOW>;
-		};
-
-		client_red {
-			label = "red:client";
-			gpios = <&led_gpio 3 GPIO_ACTIVE_LOW>;
-		};
-
-		client_green {
-			label = "green:client";
-			gpios = <&led_gpio 2 GPIO_ACTIVE_LOW>;
-		};
-
-		left_blue {
-			label = "blue:left";
-			gpios = <&led_gpio 1 GPIO_ACTIVE_LOW>;
-		};
-
-		right_blue {
-			label = "blue:right";
-			gpios = <&led_gpio 0 GPIO_ACTIVE_LOW>;
-		};
-	};
-
-	keys {
-		compatible = "gpio-keys";
-
-		reset {
-			label = "Reset button";
-			linux,code = <KEY_RESTART>;
-			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
-			debounce-interval = <60>;
-		};
-
-		wps {
-			label = "WPS button";
-			linux,code = <KEY_WPS_BUTTON>;
-			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
-			debounce-interval = <60>;
-		};
-
-		extender_apmode {
-			label = "EXTENDER/APMODE switch";
-			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
-			linux,code = <BTN_0>;
-			linux,input-type = <EV_SW>;
-			debounce-interval = <60>;
-		};
-	};
-};
-
-&pcie {
-	status = "okay";
-
-	wifi@0,0 {
-		compatible = "qcom,ath10k";
-		reg = <0 0 0 0 0>;
-
-		nvmem-cells = <&macaddr_artmtd_c>, <&precal_art_5000>;
-		nvmem-cell-names = "mac-address", "pre-calibration";
-	};
-};
-
-&pll {
-	clocks = <&extosc>;
-};
```

Comment:
Why did you remove the `pll`node?

Reply:
This is redundant. We have already defined it in `qca956x.dtsi`. I did a lot of cleaning in #10925.
https://github.com/openwrt/openwrt/blob/cb24be47ffda12616a0670e9d29a96f9530a19b8/target/linux/ath79/dts/qca956x.dtsi#L97